README_CONTENT.rst

```diff
@@ -0,0 +1,55 @@
+
+
+``HookMan`` is a python package that provides to applications a plugin management system, 
```

Comment:
Suggestion:

```rst
``HookMan`` is a python package that provides a plugin management system to applications, specially those who are written (in totally or partially) in C++. It enables external contributors to implement plugins which act as extensions written in C/C++ that interact with the application through well-defined *hooks*.

This system was largely inspired by `pluggy <https://github.com/pytest-dev/pluggy>`_, the plugin system which powers `pytest <https://github.com/pytest-dev/pytest>`_, `tox <https://github.com/tox-dev/tox>`_, and `devpi <https://github.com/devpi/devpi>`_, but with the intent to be called from a C++ application rather than from Python.
```

Reply:
What do you think about the second paragraph? Should I keep it?
  
    It was conceived to facilitate the application development, allowing hooks to be exposed in a
    clear way and allowing plugins to be developed without access to classes or data from the 
    application.